### ISSUE TITLE
PW intake: Add Spanish toggle to new R&S pages from CAIA

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#92454]
## Statuses
**[2024-09-16] [Fran]** Two URLs approved; the other two were crossed out, and Randi H will create a new ticket for those.
**[2024-09-09] [Fran]** Awaiting final QA/approval to move forward with this. Moving to Next refinement so it'll be refined when approval is given. Translated left nav and breadcrumb text requested.

### Indicate if you need front-end engineering support for

- [ ] Content audit
- [ ] Implementation of REACT sign-in widgets
- [ ] Support for UAT testing or phased roll out of app/tool landing page
- [X] Implementation of Translation widgets
- [ ] Something else

### Describe your request

We recently published in Spanish 4 new R&S articles related to sign-in. We need to get the Spanish toggles added to these. There is no left nav for these. The new pages are the following:

- https://www.va.gov/resources/signing-in-to-vagov-esp/
- ~~https://www.va.gov/resources/creating-an-account-for-vagov-esp/~~
- https://www.va.gov/resources/what-to-bring-to-create-your-online-sign-in-account-esp/
- ~~https://www.va.gov/resources/verifying-your-identity-on-vagov-esp/~~

[Fran] Per Randi H, the items crossed out aren't ready yet. She'll create a new ticket for those.

### Time constraints / launch deadlines

As sign-in is a high-priority area we are hoping to get these up ASAP. However, we are still waiting for final QA back from State. We are hoping to get that in the next few days and then we would be ready to go live with the toggles.

### Your product/team Slack channel

Sitewide CAIA

### Your OCTODE product owner

Randi Hecht

### Notify Sitewide Public Websites of this issue, and tag @Fran Cross & @jilladams.

- [X] I acknowledge that I must notify `#sitewide-public-websites` in Slack after submitting this issue.

**Comment by FranECross** [comment]
@RLHecht We'll need the translated text for the left nav/breadcrumbs to ensure we implement what's expected. Thanks in advance!

**Comment by RLHecht** [comment]
Just realized that since these are in resources and support, there won't be any left nav work. The breadcrumbs should match the H1. Here's an example of another page we did like this: https://www.va.gov/resources/the-pact-act-and-your-va-benefits-esp/

**Comment by jilladams** [comment]
Moved to top of Next Refinement to make sure we discuss this week in refinement / queue for Sprint 12, given the "asap" request.

**Comment by RLHecht** [comment]
FYI we are good to go on our end with these two:

- https://www.va.gov/resources/signing-in-to-vagov-esp/
- https://www.va.gov/resources/what-to-bring-to-create-your-online-sign-in-account-esp/

The other two pages have content changes we'll need to make before we can have the toggle added.

**Comment by FranECross** [comment]
@RLHecht Should a new ticket be created for the two that aren't ready yet, so that we can get the two that are ready into the sprint starting next Wednesday?

**Comment by RLHecht** [comment]
That's fine with me. Do you want me to create a new one or do you want to take the two out that can't be worked on yet? Just let me know how it's easiest to track.

**Comment by JonathanKamensDVA** [comment]
Hi @FranECross! Please note that the "Awaiting Approval" tag is specifically for access requests awaiting approval from OCTO. Please don't use it for other things. Thanks.

**Comment by randimays** [comment]
Language toggles have been added to these pages:

- https://www.va.gov/resources/signing-in-to-vagov/
- https://www.va.gov/resources/signing-in-to-vagov-esp/
- https://www.va.gov/resources/what-to-bring-to-create-your-online-sign-in-account/
- https://www.va.gov/resources/what-to-bring-to-create-your-online-sign-in-account-esp/


Validated successfully in production. Closing as complete.
